### DISCUSSION TITLE
How to manage multiple orders in NFT Detail

### DISCUSSION THREAD

**Discussion by taijusanagi** [kaguraswap/monorepo#124] [Ideas]
Current implementation is 

<img width="1415" alt="Screen Shot 2022-07-08 at 12 43 22" src="https://user-images.githubusercontent.com/38043569/177912620-8761bbbb-4dd0-43ff-8955-046204fc429b.png">

and now we are building a function to display only the proper button conditionally.

https://github.com/kaguraswap/monorepo/issues/94

But I want to have new implementation pattern.

**Comment by taijusanagi** [comment]
An above implementation is an nft-based approach.

but we can consider order based approach too.

<img width="787" alt="Screen Shot 2022-07-08 at 12 48 13" src="https://user-images.githubusercontent.com/38043569/177913092-6d26a8b3-3b50-4175-a3db-1acee2b9e2cc.png">

**Comment by taijusanagi** [comment]
Benefit of this approach is 

- since we are building a multi-platform NFT market (with aggregator in the future) using seaport, 0x, Zora. Order-based approach is easy to support this.

- seaport has function to exhange NFT to NFT and bulk swap too. Order-based approach is easy to support this.

**Comment by ksuhara** [comment]
my only concern is that users are familiar with OpenSea's UI/UX and something different from it would cause confusion.

besides that, i like the idea and sounds suitable for iframe embed uses

**Reply by taijusanagi** [reply]
We can add OpenSea's UI/UX later. 

But I think is better to focus on experimental functions.

**Comment by taijusanagi** [comment]
Let's take order based-approach first.

**Comment by ksuhara** [comment]
for the NFT page, list like this will appear, right?
<img width="1360" alt="スクリーンショット 2022-07-08 13 04 29" src="https://user-images.githubusercontent.com/45842611/177914809-a1accc70-e558-4b0e-aca8-06be9d23786b.png">
